Koha/REST/V1/Holding.pm

```diff
@@ -35,6 +35,30 @@ use Try::Tiny;
 
 sub list {
     my $c = shift->openapi->valid_input or return;
+
+    return try {
+        my $filter;
+        my $args = $c->req->params->to_hash;
+
+        unless ($args) {
+            return $c->render( status  => 400,
+                openapi => { error => "At least one query parameter is required" } );
+        }
+
+        for my $filter_param ( keys %$args ) {
+            $filter->{$filter_param} = { LIKE => $args->{$filter_param} . "%" };
```

Comment:
If the user only wants exact match, this is computationally inefficient in the DB.
There are potentially tens of millions of holdings records in the DB.
A fast access should be the default with optional fuzzy matching.

Reply:
Starts-with matching is now only for date and datetime fields.